cmd/upload.go

```diff
@@ -80,6 +80,11 @@ func init() {
 					progressBar.Add(1)
 				}
 
+				if branch == "" && git.Branch == "" {
```

Comment:
Should we _require_ branch information? That doesn't seem very friendly, especially without a default.

Reply:
It's now optional as long as there's no associated commit